vpr/src/place/feasible_region_move_generator.cpp

```diff
@@ -33,6 +33,7 @@ e_create_move FeasibleRegionMoveGenerator::propose_move(t_pl_blocks_to_be_moved&
 
     /* Calculate the feasible region */
     t_pl_loc to;
+    to.layer = from.layer;
```

Comment:
Maybe comment:  just keeping the blocks on their current layer, but maybe in the future the feasible region computation should try to figure out a good layer.